documentation.tex

```diff
@@ -0,0 +1,281 @@
+
+Since \texttt{refsheet} is based on \texttt{article} most options can be passed on.
+
+\section{Additional Options}
+
+%%% Version 0 style
+% 
+% \begin{ttdesc}[labelwidth=\widthof{\texttt{margin=<length>}}]
+% \item[rscols=<num>] How many columns, default is 3.
+% \item[margin=<length>] Passed to \texttt{geometry}.
+% \end{ttdesc}
+% 
+%%%
+
+%%% Version 1 style
+%
+\begin{rslisttt}{margin=<length>}
+  \item[rscols=<num>] How many columns, default is 3.
+  \item[margin=<length>] Passed to \texttt{geometry}, default 1~cm.
+\end{rslisttt}
+
+\section{Disabled Options}
+
+\begin{rslisttt}{twocolumn}
+\item[portrait]  only supports \texttt{landscape}; ignored, warning.
+\item[titlepage] saving space, so no title page; ignored, warning.
+\item[twocolumn] class uses \texttt{multicol}; break, error.
+\end{rslisttt}
+
+\section{Features}
+
+\begin{itemize}
+\item Dense settings to save maximum space.
+\item No enumeration of \texttt{(sub)section}s.
+\item Customised itemise environments.
+\end{itemize}
+
+\section{Environments}
+
+\subsection{Description based environments}
+
+All of the environments below have a mandatory argument, 
+which determines the width of the description label. 
+Therefore it should contain the longest lable which is in use.
+
+\begin{rslisttt}{rslisttt}
+\item[rslist]    Description based list, normal text
+\item[rslisttt]  dito, label in \texttt{truetype}
+\item[rslistbf]  dito, label in \textbf{bold}
+\item[rslistit]  dito, label in \textit{italics}
+\end{rslisttt}
+
+The above is achieved with the following (abbreviated) code.
+
+\begin{lstlisting}
+\begin{rslisttt}{rslisttt}
+\item[rslist]    Description based [...]
+\item[rslisttt]  dito, label in \texttt{truetype}
+\item[rslistbf]  dito, label in \textbf{bold}
+\item[rslistit]  dito, label in \textit{italics}
+\end{rslisttt}
+\end{lstlisting}
+
+\subsection{Inline description list}
+
+With the environment \texttt{rsinline} the list will be
+set as a single paragraph.
+It takes one optional argument for the font of the label:
+\begin{rsinline}%{\ttfamily}
+\item[\mybackslash{}bfseries] \textbf{bold};
+\item[\mybackslash{}itshape]  \textit{italics};
+\item[\mybackslash{}ttfamily] \texttt{truetype},
+\end{rsinline}
+which is the default if omitted.
+
+A similar list like the above can be created with the following code.
+
+\begin{lstlisting}
+One optional argument:
+\begin{rsinline}{\ttfamily}
+\item[bfseries] \textbf{bold};
+\item[itshape]  \textit{italics};
+\item[ttfamily] \texttt{truetype},
+\end{rsinline}
+which is the default.
+\end{lstlisting}
+
+\subsection{Column assorted list}
+
+The environment \texttt{rscolslist} provides acess to a list, 
+which is set in multiple columns.
+The number of these can be given as an optional argumnet,
+the default is to use two.
+
+The following example produces the list below it.
+
+\begin{lstlisting}
+\begin{rscolslist}[3]
+\item[\( \times \)]   \lstinline|\times| 
```

Comment:
Please use `$` here, too (see below)

Reply:
I think this is a personal choice, and I have a preference for the latex style. There are no readability issues with that listing and the output will be the same. 

The other arguments from before are of lesser concern here, because in the current state of the definition, it is unlikely to be used wrongly.